test/functional/glance_controller_test.go

```diff
@@ -239,13 +244,13 @@ var _ = Describe("Glance controller", func() {
 			mariadb.SimulateMariaDBDatabaseCompleted(glanceTest.Instance)
 			th.SimulateJobSuccess(glanceTest.GlanceDBSync)
 			keystone.SimulateKeystoneServiceReady(glanceTest.Instance)
-			keystone.SimulateKeystoneEndpointReady(glanceTest.GlanceExternal)
+			keystone.SimulateKeystoneEndpointReady(glanceTest.GlanceSingle)
 		})
 		It("Creates glanceAPI", func() {
 			// Default type is "split", make sure that behind the scenes two
 			// glanceAPI deployment are created
-			GlanceAPIExists(glanceTest.GlanceExternal)
-			GlanceAPIExists(glanceTest.GlanceInternal)
+			GlanceAPIExists(glanceTest.GlanceSingle)
+			//GlanceAPIExists(glanceTest.GlanceInternal)
```

Comment:
Should the comment here by changed to reflect the test code changes?

Reply:
@abays thank you for the review! So yeah, good catch, I'm going to update it in my next update.